### PR TITLE
Fix(Toolbar): Add tests for toolbar rendering

### PR DESCRIPTION
**Pull request by betatim**
Checks that the menu entry changes for different types of cells.

### CODE REVIEW

File: test/renderer/components/cell/toolbar-spec.js [hunk 1]
```diff
@@ -30,6 +30,19 @@ describe('Toolbar', () => {
...
+    const toolbar = mount(
+      <Toolbar type={'markdown'} />
+    );
+    expect(toolbar.text()).to.contain('Convert to Code Cell');
+  });
+  it('shows "convert to markdown cell" menu entry for code type', () => {
+    const toolbar = mount(
+      <Toolbar type={'code'} />
+    );
+    expect(toolbar.text()).to.contain('Convert to Markdown Cell');
```
Comment (rgbkrk) [inline test/renderer/components/cell/toolbar-spec.js:43]: This handles the case of an initial render, you'll still want to do a `setProps` after this first expect.

```js
const toolbar = mount(
  <Toolbar type={'code'} />
);
expect(toolbar.text()).to.contain('Convert to Markdown Cell');

toolbar.setProps({ type: 'markdown' })
expect(toolbar.text()).to.contain('Convert to Code Cell');
```

Comment (betatim) [reply test/renderer/components/cell/toolbar-spec.js:43]: True dat!

File: test/renderer/components/cell/toolbar-spec.js [hunk 2]
```diff
@@ -30,6 +30,26 @@ describe('Toolbar', () => {
...
+  });
+  it('changes "Convert to ..." menu entry on type change', () => {
+    const toolbar = mount(
+      <Toolbar type={'code'} />
+    );
+    expect(toolbar.text()).to.contain('Convert to Markdown Cell');
+    toolbar.setProps({ type: 'markdown' });
+    expect(toolbar.text()).to.contain('Convert to Code Cell');
```
Comment (rgbkrk) [inline test/renderer/components/cell/toolbar-spec.js:51]: 👍

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
These are some quality tests ensuring proper behavior, much thanks @betatim!

**Comment by betatim** [comment]
:beers:

**Pull request merged**